pkg/agent/agent_test.go

```diff
@@ -274,23 +313,31 @@ func TestListReleases(t *testing.T) {
 			status:    "all",
 			listLimit: defaultListLimit,
 			releases: []releaseStub{
-				releaseStub{"wordpress", "default", 1, release.StatusDeployed},
-				releaseStub{"wordpress", "dev", 2, release.StatusUninstalled},
+				releaseStub{"wordpress", "default", 1, "1.0.0", release.StatusDeployed},
+				releaseStub{"wordpress", "dev", 2, "1.0.0", release.StatusUninstalled},
 			},
 			expectedApps: []proxy.AppOverview{
 				proxy.AppOverview{
 					ReleaseName: "wordpress",
 					Namespace:   "default",
-					Version:     "1",
+					Version:     "1.0.0",
 					Status:      "deployed",
 					Icon:        "https://example.com/icon.png",
+					ChartMetadata: chartv1.Metadata{
+						Version: "1.0.0",
+						Icon:    "https://example.com/icon.png",
+					},
 				},
 				proxy.AppOverview{
 					ReleaseName: "wordpress",
 					Namespace:   "dev",
-					Version:     "2",
+					Version:     "1.0.0",
```

Comment:
Is this change trivially correct? If not, maybe add a note in the PR?

Reply:
In this case it's not needed (the version is not relevant either)